src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -25,6 +26,7 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
     line3: 'If you wait a year before applying then you will need to explain the delay to the court.',
   },
   checkboxLine: `I want to ${isDivorce ? 'finalise my divorce' : 'end my civil partnership'}`,
+  continue: 'Submit',
```

Comment:
you might not need this as it might be in common.content.ts

Reply:
It's present in common, but the text doesn't match the google doc.

Reply:
fair, i think we should have a submit one as it seemed to be used in other places but its not a big deal.